src/librustc_mir/diagnostics.rs

```diff
@@ -1585,12 +1592,15 @@ Or:
 ```
 struct Value {}
 
+fn borrow(val: &Value) {}
+
 fn eat(val: Value) {}
 
 fn main() {
     let x = Value{};
     {
         let _ref_to_val: &Value = &x;
+        borrow(_ref_to_val);
     }
```

Comment:
Should I keep this set of braces in?
It is one way to resolve the error.

Reply:
Sorry, I forgot that we still have AST borrow check on the 2015 edition. Keep them for now.

Reply:
I can remove the braces from all the examples except this one, and they give the expected results for both 2015 and 2018.

erroneous: Fails
Take by ref: Passes
Implement Copy: Passes

Reply:
Ok, it's fine like this for now.